helm/db-controller/values.yaml

```
@@ -132,8 +132,8 @@ controllerConfig:
   athena-shared:
     masterUsername: root
   authSource: secret
-  # if aws authorization is used iam role must be provided
-  #iamRole: rds-role
+  #aws or gcp
+  cloud: aws
```

This isn't configured in DC. TODO to fix this in a later PR
```suggestion
  # FIXME: change me to `cloud_provider` and update DC kubevela workflow to populate the field
  # https://github.com/Infoblox-CTO/deployment-configurations/pull/61446/files#diff-796b260c9364c2558037d151b9450f8b72904b41ccb90b2f1da7aff36e97e141R9-R24
  cloud: aws
```